src/interfaces/IMetaMorpho.sol

```diff
@@ -86,8 +96,21 @@ interface IMetaMorpho is IERC4626 {
     function reallocate(MarketAllocation[] calldata withdrawn, MarketAllocation[] calldata supplied) external;
 }
 
-interface IPending {
-    function pendingTimelock() external view returns (PendingUint192 memory);
-    function pendingCap(Id) external view returns (PendingUint192 memory);
+/// @dev This interface is inherited by MetaMorpho so that signatures are checked by the compiler.
```

Comment:
```suggestion
/// @dev This interface is inherited by MetaMorpho so that function signatures are type-checked by the compiler.
```

Reply:
Strictly speaking the function signatures are the types, they are not type-checked, it's the combination of a function and the function signature that is type-checked. I think it's fine to just say "signatures are checked", it's understandable like that

Reply:
So a more appropriate suggestion would be:

```suggestion
/// @dev This interface is inherited by MetaMorpho so that functions are type-checked by the compiler.
```

Reply:
I'm not blocking, but I raise the fact that my eyes blinked at "signatures" to understand what type of signature we were talking about

Reply:
I specified "function signatures", re-open if it's still unclear